src/endpoints/achievements.ts

```diff
@@ -0,0 +1,164 @@
+import { AbstractEndpoint } from '../endpoint'
+import { Schema } from './schemas/schema'
+
+
+type Flag = 'PvE' | 'PvP' | 'WvW' | 'SpecialEvent'
+type Product = 'GuildWars2' | 'HeartOfThorns' | 'PathOfFire' | 'EndOfDragons'
+type Condition = 'HasAccess' | 'NoAccess'
+type LevelTuple = [number, number]
+type LevelObj = { min: number, max: number }
+type RequiredAccess = { product: Product, condition: Condition }
+
+export namespace SchemaOld {
+  /** {@link https://wiki.guildwars2.com/wiki/API:2/achievements/daily} */
+  type Daily = {
+    id: number,
+    level: LevelObj,
+    required_access: Product[]
+  }
+  
+  /** {@link https://wiki.guildwars2.com/wiki/API:2/achievements/daily/tomorrow} */
+  export interface Dailies {
+    pve: Daily[],
+    wvw: Daily[],
+    fractals: Daily[],
+    special: Daily[]
+  }
+
+  /** {@link https://wiki.guildwars2.com/wiki/API:2/achievements/categories} */
+  export interface Category {
+    id: number,
+    name: string,
+    description: string,
+    order: number,
+    icon: string,
+    achievements: number[],
+  }
+}
+
+// If using schema 2019-05-16T00:00:00.000Z or later 
+export namespace SchemaNew {
+  /** {@link https://wiki.guildwars2.com/wiki/API:2/achievements/daily} */
+  type Daily = {
```

Comment:
I think it makes sense to use `interface` for all of these, just for consistency's sake.

Reply:
Absolutely! That one and probably a few more slipped past me. Will make sure all structured types are interfaces.